src/Twilio/Http/Request.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
```

Comment:
Is this needed?

Reply:
Ugh darn, remnant from local testing. nice catch, thanks.